microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,225 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+<!-- ====================================================================== -->
+## 1.0.1587.40
+
+Release Date: Feb 13, 2023  
+
+[NuGet package for WebView2 SDK 1.0.1587.40](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1587.40)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 110.1587.40 or higher. 
+
+### General
+
+#### Promotions
+
+The following items are now stable:
+
+*  Additional options used to create WebView2 Environment to manage custom scheme registration.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2EnvironmentOptions Class](#)
+   * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](#)
+* [CoreWebView2CustomSchemeRegistration Class](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2EnvironmentOptions Class](#)
+* [CoreWebView2CustomSchemeRegistration Class](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2EnvironmentOptions4 interface](#)
+   * [ICoreWebView2EnvironmentOptions4::GetCustomSchemeRegistrations method](#)
+   * [ICoreWebView2EnvironmentOptions4::SetCustomSchemeRegistrations method](#)
+* [ICoreWebView2CustomSchemeRegistration interface](#)
+   * [ICoreWebView2CustomSchemeRegistration::GetAllowedOrigins method](#)
+   * [ICoreWebView2CustomSchemeRegistration::SetAllowedOrigins method](#)
+   * [ICoreWebView2CustomSchemeRegistration::HasAuthorityComponent property (get](#), [put)](#)
+   * [ICoreWebView2CustomSchemeRegistration::SchemeName property (get)](#)<!--no put-->
+   * [ICoreWebView2CustomSchemeRegistration::TreatAsSecure property (get](#), [put)](#)

```

Comment:
```suggestion
* [ICoreWebView2CustomSchemeRegistration interface](/microsoft-edge/webview2/reference/win32/icorewebview2customschemeregistration?view=webview2-1.0.1518.46&preserve-view=true)
   * [ICoreWebView2CustomSchemeRegistration::GetAllowedOrigins method](/microsoft-edge/webview2/reference/win32/icorewebview2customschemeregistration?view=webview2-1.0.1518.46&preserve-view=true#getallowedorigins)
   * [ICoreWebView2CustomSchemeRegistration::SetAllowedOrigins method](/microsoft-edge/webview2/reference/win32/icorewebview2customschemeregistration?view=webview2-1.0.1518.46&preserve-view=true#setallowedorigins)
   * [ICoreWebView2CustomSchemeRegistration::get_HasAuthorityComponent method](/microsoft-edge/webview2/reference/win32/icorewebview2customschemeregistration?view=webview2-1.0.1518.46&preserve-view=true#get_hasauthoritycomponent)
   * [ICoreWebView2CustomSchemeRegistration::put_HasAuthorityComponent method](/microsoft-edge/webview2/reference/win32/icorewebview2customschemeregistration?view=webview2-1.0.1518.46&preserve-view=true#put_hasauthoritycomponent)
   * [ICoreWebView2CustomSchemeRegistration::get_SchemeName method](/microsoft-edge/webview2/reference/win32/icorewebview2customschemeregistration?view=webview2-1.0.1518.46&preserve-view=true#get_schemename)<!--no put-->
   * [ICoreWebView2CustomSchemeRegistration::get_TreatAsSecure method](/microsoft-edge/webview2/reference/win32/icorewebview2customschemeregistration?view=webview2-1.0.1518.46&preserve-view=true#get_treatassecure)
   * [ICoreWebView2CustomSchemeRegistration::put_TreatAsSecure method](/microsoft-edge/webview2/reference/win32/icorewebview2customschemeregistration?view=webview2-1.0.1518.46&preserve-view=true#put_treatassecure)
```
experimenting with separate items for get "method" and put "method" to reduce manual modifications